### PR TITLE
New Slack Mac Cheat Sheet

### PR DESCRIPTION
**Pull request by mayank7**
New Slack Cheat Sheet containing keyboard shortcuts for Mac.

IA Page: https://duck.co/ia/view/slack_mac_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [Slack Mac Cheat Sheet](https://duck.co/ia/view/slack_mac_cheat_sheet)

**Description**: Mac Keyboard shortcuts for Slack

**Example Query**: [https://duckduckgo.com/?q=slack+mac+cheat+sheet](https://beta.duckduckgo.com/?q=https://duckduckgo.com/?q=slack+mac+cheat+sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**: https://get.slack.help/hc/en-us/articles/201374536-Slack-keyboard-shortcuts

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/slack_mac_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/slack_mac_cheat_sheet)*

File: share/goodie/cheat_sheets/json/slack-mac.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
+{
+  "id": "slack_cheat_sheet_mac",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/slack-mac.json]: whoops, this should be `slack_mac_cheat_sheet` to match the IA Page

File: share/goodie/cheat_sheets/json/slack.json [hunk 2]
```diff
@@ -107,7 +107,7 @@
     ],
     "Message shortcuts": [
       {
-        "key": "[↑] in empty messag field",
+        "key": "[↑] in empty message field",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/slack.json]: Updates/fixes should be in a separate PR. When adding a new IA, only the required files should be added in the Pr

**Comment by mayank7** [comment]
@moollaza I changed the id to slack_mac_cheat_sheet and removed updates in other file

**Comment by moollaza** [comment]
@mayank7 for your IA Pages, the `example queries` should just be the plain query a user would search, not a link. We'll create a link out of the word. In this case the first example query should be `slack mac cheat sheet`

**Comment by mayank7** [comment]
@moollaza Sorry, I will take care of it now.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Slack Mac Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/slack_mac_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/slack-mac.json [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+      {
+        "key": "[↑] in empty messag field",
+        "val": "Edit your last message in the current channel"
+      },
+      {
+        "key": "[Page Up] or [Page Down], [Home] or [End]",
+        "val": "Scroll through your messages (toggle this in Preferences)"
+      },
+      {
+        "key": "[Tab]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/slack-mac.json]: You can remove the `[]` here

**Comment by moollaza** [comment]
@mayank7 Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@mayank7 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!
